objects/Fan-MadeAccessories.aa8b38/CleanUpHelper.26cf4b.ttslua

```diff
@@ -249,6 +254,11 @@ function getTrauma()
     end
 end
 
+-- get rid of bless/curse tokens
+function removeBlurse()
+    if blurse ~= nil then blurse.call("doRemove", "White") end
```

Comment:
Is this dependent on the new Bless/Curse Token Manager? I still see the old version in the toolupdates branch.

Reply:
Yes, it is. I am a git novice and don't know how to mark this as dependant 😅